custom-nb-image/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 FROM  quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8-4c8f26e
-
+ARG SDK_VERSION=0.4.4
```

Comment:
Can we change the default to `0.*` so that we get the most up to date version if none is specified? (We will need to update this once a `1.0` comes out at some point in the future.